posthog/models/team/team.py

```diff
@@ -223,7 +223,6 @@ class CookielessServerHashMode(models.IntegerChoices):
 
 
 class SessionRecordingRetentionPeriod(models.TextChoices):
-    LEGACY = "legacy", "Legacy Retention"
```

Comment:
will self-hosted all have been resolved by previous releases so they don't have legacy as an option?

(otherwise we could instead put in a validation that legacy isn't allowed where `isCloudOrDev`)

Reply:
Good point - added a script to the migration to move all teams away from 'legacy' before altering the schema...